source/how-tos/support_ticket.rst

```diff
@@ -0,0 +1,132 @@
+* ``from``: The email address to set the email from field when sending the email. It defaults to the email field set in the form.
+  This is useful to prevent the incoming emails being considered spam.
+* ``to``: The destination email address of your Help Desk. 
+* ``delivery_method``: Rails setting to set the transport mechanism to use. This is usually ``smtp``.
+  For more information see the `Rails documentation on mailers <https://guides.rubyonrails.org/action_mailer_basics.html#action-mailer-configuration>`_.
+* ``delivery_settings``: Rails settings object to set the transport configuration properties.
+  See example below or the `Rails documentation on mailers <https://guides.rubyonrails.org/action_mailer_basics.html#action-mailer-configuration>`_.
+
+Sample configuration:
+
+  .. code-block:: yaml
+
+    support_ticket:
+      email:
+        # Optional.
+        # Text/HTML added to the page under the support ticket header
+        description: |
+          <strong>My Description Text</strong><br />
+          This is a description text for the frontend with HTML support
+        # Optional.
+        # To set the email from field. Defaults to the email set in the form.
+        from: "config@example.com" 
+        # Required.
+        # Your support system email address. The email will be sent to this address
+        to: "support@support.system.com"
+        # delivery_method and delivery_settings are Rails configuration items
+        # that can be set in this file to simplify configuration.
+        # If you prefer, you can configure Rails in the usual way.
+        delivery_method: "smtp"
+        delivery_settings:
+          address: 'smtp.gmail.com'
+          port: 587
+          domain: 'example.com'
+          user_name: '<username>'
+          password: '<password>'
+          authentication: 'plain'
+          enable_starttls_auto: true
+          open_timeout: 5
+          read_timeout: 5
+
+Customizations
+..............
+
+The support ticket form can be customized using the ``form`` and ``attributes`` configuration properties.
+In the same way that you can configure the :ref:`form for interactive applications <enable-desktops-modify-form-attributes>`,
+the support ticket form fields can be arranged or changed as required.
+
+.. note::
+  
+  Please note that an ``email`` field must be present in the form for the support ticket feature to be funtional.
+
+Custom fields can be added, but they will require a custom email template to make use of the field values to generate the email body.
+Drop the template file into ``/etc/ood/config/apps/dashboard/views/support_ticket/email`` and reference it
+in the configuration. This template file can be any format Rails recognizes for emails, usually the ``.txt.erb`` extension.
+
+The example below shows a custom template configuration with 3 fields and how to use them in the template to generate the email body.
```

Comment:
Can you also mention here that this is the same structure as an interactive app's form?

Reply:
I mentioned that form updates follow the same structure as interactive apps at the beginning of the `Customizations` section. Shall I repeat it here as well?

Reply:
nope, sorry, disregard! I misread that bit.